### PR TITLE
Bowtie2 + bionode-ncbi

### PR DESCRIPTION
**Pull request by bfrgoncalves**

### CODE REVIEW

**Review by miguelpmachado — CHANGES_REQUESTED** [review]
Update README

File: bionode-ncbi/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#DOCKERFILE FOR BIONODE-NCBI
+FROM ubuntu:16.04
+MAINTAINER Bruno Gonçalves, bfgoncalves@medicina.ulisboa.pt
```
Comment (miguelpmachado) [inline bionode-ncbi/Dockerfile:3]: E-mail should be <e-mail@something.pt>

**Review by miguelpmachado — CHANGES_REQUESTED** [review]

File: bionode-ncbi/Dockerfile (hunk 1 above)
Comment (bfrgoncalves) [reply bionode-ncbi/Dockerfile:3]: Already changed it

**Review by miguelpmachado — CHANGES_REQUESTED** [review]
Update master README

**Review by miguelpmachado — APPROVED** [review]

**Pull request merged**